tests/yam/agama/generate_dud.pm

```diff
@@ -0,0 +1,31 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run interactive installation with Agama,
+# using a web automation tool to test directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use strict;
+use warnings;
+use testapi;
+use utils;
+use autoyast qw(expand_agama_profile generate_json_profile);
+
+sub run {
+    my $self = shift;
+    my $dud = get_required_var('DUD');
+    my $profile_url = get_required_var('INST_AUTO');
+
+    select_console 'install-shell';
+
+    zypper_call("ar -f -G https://download.opensuse.org/repositories/home:/snwint:/ports/SLFO-Main/home:snwint:ports.repo");
+    zypper_call("in -y mkdud");
+    assert_script_run("mkdir -p tmp/dud/root");
+    diag $profile_url;
```

Comment:
similar, diag better below line 18, would be more effective for debugging.

Reply:
Not needed, removed.